homeassistant/components/unifi/device_tracker.py

```diff
@@ -239,10 +242,13 @@ def device_state_attributes(self):
 
         attributes["is_wired"] = self.is_wired
 
-        for variable in CLIENT_STATIC_ATTRIBUTES + CLIENT_CONNECTED_ATTRIBUTES:
+        if self.is_connected:
```

Comment:
Calling `is_connected` 518 * `CLIENT_CONNECTED_ATTRIBUTES` per second added up quickly.

Reply:
Awesome! I'll review it in the afternoon